esmvaltool/diag_scripts/perfmetrics/zonal.ncl

```diff
@@ -17,9 +17,9 @@ begin
   annots = project_style(info_items, diag_script_info, "annots")
 
   ; Loop over datasets
-  do ii = 0, dimsizes(modidx) - 1
+  do modid = 0, dimsizes(modidx) - 1
 
-    imod = modidx(ii)
+    imod = modidx(modid)
```

Comment:
I think with fix #3098 (merged today!), this is not needed anymore.

Reply:
Great, it took me a while to figure out why my loop is breaking early. I guess for infinite loops this was hell.